src/zoid/fields/component.jsx

```diff
@@ -51,6 +51,30 @@ export function getFieldsComponent() : ZoidComponent<FieldsProps> {
             },
 
             props: {
+
+                createOrder: {
```

Comment:
We agreed that SPB would be handling this right? 

- User clicks on SPB
- SPB communicates with fields (can pass in order id if needed)
- SPB then launches popup